walkthroughs/howto-access-log/colorapp-ecs/fargate/README.md

```diff
@@ -0,0 +1,231 @@
+# Using AWS App Mesh with Fargate
+
+## Overview
```

Comment:
This readme is really odd in the context of the access log example

Reply:
I will reconstruct this example, not using the bug bash, it will be easier for customer to use

Reply:
Since the console changes are out, customers should be using that as the major way for accessing this feature